.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "packages/contracts-bedrock/lib/forge-std"]
 	path = packages/contracts-bedrock/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "packages/contracts-bedrock/lib/permit2"]
+	path = packages/contracts-bedrock/lib/permit2
+	url = https://github.com/lukasrosario/permit2
```

Comment:
temporarily pointing to fork on my branch until pragma downgrade gets upstreamed

Reply:
pointing at uniswap again